tests/unit/test_table.py

```diff
@@ -858,22 +858,7 @@ def test_read_retry_rows(self):
         response_iterator = _MockReadRowsIterator(response_2)
 
         # Patch the stub used by the API method.
-        data_api.table_path.return_value = (
-            "projects/"
-            + self.PROJECT_ID
-            + "/instances/"
-            + self.INSTANCE_ID
-            + "/tables/"
-            + self.TABLE_ID
-        )
-        table_api.table_path.return_value = (
```

Comment:
did you mean to drop the table_api assignment here?

Reply:
yes - some of these assignments were from when I was trying out which items to mock to get the tests to work - this one was unnecessary.